lisa/tools/ethtool.py

```diff
@@ -1108,11 +1108,12 @@ def get_device_enabled_features(
         interface_info = self.node.tools[Ip].run(interface).stdout
 
         # Example output:
-        # options=8051b<RXCSUM,TXCSUM,VLAN_MTU,VLAN_HWTAGGING,TSO4,LRO,LINKSTATE>
-        # The features are separated by comma and enclosed by "<>"
+        # features=((RXCSUM,TXCSUM,VLAN_MTU,VLAN_HWTAGGING,TSO4,LRO,LINKSTATE), "/t")
```

Comment:
It's better to update the pattern `_interface_features_pattern`, or add a new pattern to handle both cases.

Reply:
You can use some online tool debugging regexp, like https://regex101.com/.

Reply:
Oh I misunderstood this comment and the regex filter is still correct but the find group function is returning a tuple that looks like what I wrote. I will revert the comment and then check the other images for missing feature translations so there aren't any issues like Lili noted.

Reply:
You can update the pattern, so it can deal with the extra tuple.

Reply:
I will do that.